### DISCUSSION TITLE
Creating design tokens matching fluentui react library.

### DISCUSSION THREAD

**Discussion by subratb** [microsoft/fluentui-blazor#1760] [Q&A]
How would I create custom color specific design tokens like "colorPaletteBlueBackground2" in [FluentUI React](https://react.fluentui.dev/?path=/docs/theme-colors--page) using fluentui-blazor?

**Comment by vnbaaij** [comment]
You can't. At least not yet. 

We use the Fluent UI Web Components verson 2 script internally and that is exposing the Fluent desing version 1.

The tokens you are referring to are from Fluent design v2. Support for that is coming in the Web Components v3. We are working with the web components team to get a version where we can start implementing with.

**Reply by subratb** [reply]
Thank you for your quick response. When can we expect the first version of these changes? Also, what approach should we take to implement the same behavior now until we get these?

**Reply by vnbaaij** [reply]
We dont have a timeline yet. What is the behavior you want to get?
In general the approach would be to add CSS variables and us those.

**Reply by subratb** [reply]
To be able to use custom fill colors as background of elements.
